include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -189,23 +189,23 @@ uninitialized_fill_n(_ExecutionPolicy&& __exec, _ForwardIterator __first, _Size
 
 #if (_PSTL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN || _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN)
 
-oneapi::dpl::execution::parallel_policy
-get_unvectorized_policy(oneapi::dpl::execution::parallel_unsequenced_policy)
+const oneapi::dpl::execution::parallel_policy&
+get_unvectorized_policy(const oneapi::dpl::execution::parallel_unsequenced_policy&)
 {
-    return {};
+    return oneapi::dpl::execution::par;
 }
 
-oneapi::dpl::execution::sequenced_policy
-get_unvectorized_policy(oneapi::dpl::execution::unsequenced_policy)
+const oneapi::dpl::execution::sequenced_policy&
+get_unvectorized_policy(const oneapi::dpl::execution::unsequenced_policy&)
 {
-    return {};
+    return oneapi::dpl::execution::seq;
 }
 
 template <typename _ExecutionPolicy>
-_ExecutionPolicy&&
-get_unvectorized_policy(_ExecutionPolicy&& __exec)
+const _ExecutionPolicy&
+get_unvectorized_policy(const _ExecutionPolicy& __exec)
```

Comment:
what if an execution policy is passed by as rvalue reference?

Reply:
Yesterday we discussed with @rarutyun about this code and it's his preferences (as far as I understood him).
Technically it's correct form too.

Reply:
I suggested to have 

```cpp
template <typename _ExecutionPolicy>
_ExecutionPolicy&
get_unvectorized_policy(_ExecutionPolicy& __exec)
```

I am afraid that in the current form there might be issues with `device_policy`.

Let's also discuss this topic all together tomorrow.

Reply:
At this moment the function `get_unvectorized_policy` called from two places:
```C++
template <class _ExecutionPolicy, class _ForwardIterator>
oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy>
destroy(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardIterator __last)
{
    // ...get_unvectorized_policy(__exec)
}

template <class _ExecutionPolicy, class _ForwardIterator, class _Size>
oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy, _ForwardIterator>
destroy_n(_ExecutionPolicy&& __exec, _ForwardIterator __first, _Size __n)
{
    // ...get_unvectorized_policy(__exec)
}
```
I think this change, proposed by you, really is not required.